### ISSUE TITLE
RCS1213 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#471]
In the following sample M1 and C1 are flagged by RCS1213 but M2 and C2 aren't. Are these false negatives?

```csharp
namespace RCS1213_False_Negative
{
    class Program
    {
        private static void M1()
        {
        }

        private static void M2()
        {
#pragma warning disable IDE0000
#pragma warning restore IDE0000
        }

        private const int C1 = 0;
#pragma warning disable IDE0000
        private const int C2 = 0;
    }
}
```

**Comment by JosefPihrt** [comment]
fixed with 0fd278678724e0e4329fbec0ad29d7c59eb24fcf
